src/assets/locales/hi.js

```diff
@@ -162,6 +162,7 @@ export default {
     generic: "कुछ दिक्कत आ गयी कृपया दोबारा कोशिश करें",
     internet: "कृपया अपने इंटरनेट कनेक्शन की जाँच करें",
     copying: "कापी करते वक़्त दिक्कत हुई ",
+    auto_logout: "आप लॉग आउट हो चुके हैं!",
```

Comment:
"aapko logout kar dia gaya hain" would be the appropriate translation.

Reply:
I just used google translate
I think this makes sense
Anyway we'll get all these texts verified, so can park until that happens